nncf/torch/checkpoint_loading.py

```diff
@@ -228,8 +228,8 @@ def _clip_keys_without_collisions(keys: List[str], keys_to_ignore: List[str]) ->
     @staticmethod
     def _key_clipper(key: str) -> str:
         new_key = key
-        from nncf.torch.nncf_network import MODEL_WRAPPED_BY_NNCF_ATTR_NAME #pylint: disable=cyclic-import
-        clip_patterns = [MODEL_WRAPPED_BY_NNCF_ATTR_NAME + '.', 'module.', '|OUTPUT', '|INPUT']
+        from nncf.torch.nncf_network import LEGACY_MODEL_WRAPPED_BY_NNCF_ATTR_NAME #pylint: disable=cyclic-import
```

Comment:
Internal structure of the compressed model object has changed with this PR, and the checkpoint format basically changes too. This code provides the functionality of loading old-style checkpoints with a deprecation warning, urging the user to re-save their checkpoint, which will then save the checkpoint in the new format.

Reply:
Not really, because our scopes are specified in terms of structure of the original model, and this format has not been changed.